lbrynet/core/Wallet.py

```diff
@@ -738,20 +764,26 @@ def get_transaction(self, txid):
     def get_claim_metadata_for_sd_hash(self, sd_hash):
         return self._get_claim_metadata_for_sd_hash(sd_hash)
 
-    def get_balance(self):
-        return self.wallet_balance - self.total_reserved_points - sum(self.queued_payments.values())
+    def get_balance(self, address=None):
```

Comment:
It's probably worth making a separate function - like `lbrynet.core.Wallet.Wallet.get_address_balance` - since `lbrynet.core.Wallet.Wallet.get_balance` returns a figure based on the payments queued in lbrynet, which isn't exactly in sync with the live balance from lbryum. It also might be nice to optionally return the unconfirmed balance. 

You'd make a placeholder function that returns `defer.fail(NotImplementedError())` in the Wallet base class, and the real `get_address_balance` function in the LBRYumWallet class.

Then, you'd make the the new functionality accessible via the lbrynet api in `lbrynet.lbrynet_daemon.Daemon.Daemon.jsonrpc_wallet_balance`